packages/utils/src/loader/svg-loader.ts

```diff
@@ -48,19 +48,21 @@ const stripUnsafeNodes = (...elements: Node[]): void => {
  * @param response Request response to test
  * @returns Is valid SVG
  */
-const isValidResponse = (response: XMLHttpRequest | undefined): response is XMLHttpRequest => {
-  return !!response && response.status === 200
-  && response.getResponseHeader('content-type') === 'image/svg+xml';
+const isValidResponse = (response: Response | undefined): response is Response => {
+  const isSVG = !!response?.headers.get('content-type')?.startsWith('image/svg+xml');
```

Comment:
```suggestion
  const isSVG = Boolean(response?.headers.get('content-type')?.startsWith('image/svg+xml'));
```

Reply:
Please check `!!` in other changes as well

Reply:
Updated by following your suggest.